### PR TITLE
Add Bootstrap dependencies and add gridding to HTML files.

### PR DESCRIPTION
**Pull request by ganewman**
### What is a quick description of the change?

- Use BootstrapCDN to add Bootstrap and related JS files as a dependency on all HTML files.
- Rewrite HTML files to use Bootstrap gridding features to align different sections of website.

### Is this fixing an issue?

none

### Are there more details that are relevant?

none

### Check lists (check `x` in `[ ]` of list items)

- [  ] Test written/updating
- [X] Tests passing
- [X] Coding style (indentation, etc)

Please explain why any are not present, if any.

### CODE REVIEW

File: project/src/main/webapp/style.css [hunk 1]
```diff
@@ -1,48 +1,4 @@
...
-}
-
-/* Candidate Page Styling */
-
-.candidate-page-info {
-  height: 40%;
-  padding: 100px 100px;
-  width: 60%;
-}
+}
```
Comment (suparnabarua) [inline project/src/main/webapp/style.css]: nit new line

File: project/src/main/webapp/index.html [hunk 2]
```diff
@@ -17,22 +24,50 @@
...
+            <form id="list-all-input" action="/directory.html">
+              <input type="hidden" name="address" value="default"/>
+              <input type="hidden" name="listAllElections" value="true"/>
+              <input type="submit" id="submit-list-all" value="List All Elections"/>
+            </form>
+          </main>
+        </div>
+      </div>
+    </div>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity=
```
Comment (suparnabarua) [inline project/src/main/webapp/index.html:56]: You're reusing the various script srces a lot. Consider making them shared variables?

**Review by suparnabarua — CHANGES_REQUESTED** [review]

File: project/src/main/webapp/candidate.html [hunk 3]
```diff
@@ -5,6 +5,13 @@
     <meta name="Voter Central Candidate Page"
           content="The sample candidate page of Voter Central, a website
                    for centralizing voting information">
+    <meta name="viewport" 
+          content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href=
+        "https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+        integrity=
+        "sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+        crossorigin="anonymous">
```
Comment (KathyFeiyang) [inline project/src/main/webapp/candidate.html]: nit: maybe align the parameters `rel`, `integrity`, `crossorigin`

**Review by KathyFeiyang — CHANGES_REQUESTED** [review]

File: project/src/main/webapp/candidate.html [hunk 4]
```diff
@@ -19,10 +27,42 @@
...
-    <header></header>
-    <nav-bar></nav-bar>
-    <main role="main">
-      <candidate-page></candidate-page>
-    </main>
+    <div class="container">
+      <div class="row justify-content-end">
+      <div class="col-9">
+          <header></header>
+      </div>
```
Comment (kellyktan) [inline project/src/main/webapp/candidate.html]: Should this be indented?

Comment (kellyktan) [reply project/src/main/webapp/candidate.html]: Same for the other files.

**Review by kellyktan — CHANGES_REQUESTED** [review]

**Review by suparnabarua — APPROVED** [review]

File: project/src/main/webapp/index.html [hunk 5]
```diff
@@ -17,22 +26,54 @@
...
+        <div class="col">
+          <nav-bar></nav-bar>
+        </div>
+        <div class="col-9">
+          <main role="main">
+            <form id="location-input" action="/directory.html">
+              <label for="address">Enter your street address:</label>
+              <br>
+              <input type="text" id="address" name="address" placeholder="...your street address..."
+                      required/>
```
Comment (KathyFeiyang) [inline project/src/main/webapp/index.html]: nit: reduce indentation

**Review by KathyFeiyang — APPROVED** [review]
LGTM pending changes for existing comments!

**Review by tjyoung99 — APPROVED** [review]

**Review by kellyktan — APPROVED** [review]

**Pull request merged**